apiary-metastore-listener/src/test/java/com/expedia/apiary/extensions/metastore/listener/ApiarySnsListenerTest.java

```diff
@@ -62,29 +65,51 @@
 @RunWith(MockitoJUnitRunner.class)
 public class ApiarySnsListenerTest {
 
+  @Rule
+  public final EnvironmentVariables environmentVariables = new EnvironmentVariables();
+
   @Mock
   private AmazonSNS snsClient;
   @Mock
   private Configuration configuration;
   @Mock
   private PublishResult publishResult;
+  @Mock
+  private StorageDescriptor newStorageDescriptor;
+  @Mock
+  private StorageDescriptor storageDescriptor;
 
   @Captor
   private ArgumentCaptor<PublishRequest> requestCaptor;
 
   private final Table table = new Table();
   private static final String TABLE_NAME = "some_table";
   private static final String DB_NAME = "some_db";
+  private static final String TABLE_LOCATION = "s3://table_location";
+  private static final String NEW_TABLE_LOCATION = "s3://table_location_1";
+
+  private static final String PARTITION_LOCATION = "s3://table_location/partition_location=2";
+  private static final String OLD_PARTITION_LOCATION = "s3://table_location/partition_location=1";
+
   private static final List<String> PARTITION_VALUES = ImmutableList.of("value_1", "1000", "value_2");
   private static final List<String> NEW_PARTITION_VALUES = ImmutableList.of("value_3", "2000", "value_4");
 
+  private static final Map<String, String> PARAMETERS = ImmutableMap
+      .of("HIVE.VERSION", "2.3.4-amzn-0", "HIVE_METASTORE_TRANSACTION_ACTIVE", "false", "HIVE.METASTORE.URIS",
+          "thrift://test_uri:9083", "HKAAS_ENABLED", "true", "HKAAS_EXPIRY_DAYS", "5");
+
   private List<FieldSchema> partitionKeys;
   private ApiarySnsListener snsListener;
 
   @Before
   public void setup() {
+    environmentVariables.set("TABLE_PARAM_FILTER", "HKAAS.*");
```

Comment:
again, will just be confusing make more generic

Reply:
removed.